docs/virtualkeyboard/index.html

```diff
@@ -189,17 +250,17 @@ <h2>The <dfn>VirtualKeyboard</dfn> Interface</h2>
         <h2>Extensions to the <dfn>ElementContentEditable</dfn> mixin</h2>
         <pre class="idl">
             partial interface ElementContentEditable {
-                [CEReactions, Reflect, ReflectOnly=("auto","manual")] attribute DOMString virtualKeyboardPolicy;
+                [CEReactions] attribute DOMString virtualKeyboardPolicy;
```

Comment:
Why remove the IDL reflection rules?

Reply:
I was getting ReSpec errors so the recommendation (per [this ](https://github.com/w3c/editing/pull/317#issuecomment-885946703)comment) is to just mention it in the definition of the attribute and link it to the IDL. In my case I mentioned it in the `virtualKeyboardPolicy` definition:
> The virtualKeyboardPolicy is an enumerated attribute whose keywords are the string, auto, and manual. The IDL attribute must reflect the respective content attributes of the same name.